### PR TITLE
Added Linux installation guide

### PR DESCRIPTION
**Pull request by Kourchenko**

### CODE REVIEW

**Comment by noahbrenner** [comment]
This might benefit from some more detail. I'm running it on XUbuntu and I don't use `sudo` to run `docker-compose`. I'll look up the info on that now.

**Comment by noahbrenner** [comment]
This setup step is optional and *does* have security impacts that need to be considered, but here are the instructions for allowing `docker-compose` to be run without being root: https://docs.docker.com/install/linux/linux-postinstall/

Perhaps include this link as well (either with the caveats or leaving that to the warning on the page itself)?

**Comment by ryanrolds** [comment]
Ubuntu 16.04 is EOLed correct? Please find instructions for Ubuntu 18.04. As this is very specific information, it may be better to put it in the Troubleshooting section.

File: README.md [hunk 1]
```diff
@@ -132,6 +132,15 @@ To resolve this, rebuild the Docker image for that specific process. For example
 $ docker-compose up --build frontend
 ```
 
+### Running Docker on Linux
+Installation on [Linux](https://askubuntu.com/questions/938700/how-do-i-install-docker-on-ubuntu-16-04-lts).
```
Comment (ryanrolds) [inline README.md]: Linux != Ubuntu. The official install instructions for Ubuntu, https://docs.docker.com/install/linux/docker-ce/ubuntu/.

File: README.md [hunk 2]
```diff
@@ -132,6 +132,15 @@ To resolve this, rebuild the Docker image for that specific process. For example
...
+
+Running the environment locally on a Linux-based OS could require running `docker-compose` commands as super user, `sudo docker-compose [commands]`.
+
+[Here is a guide for managing Docker as a non-root, user](https://docs.docker.com/install/linux/linux-postinstall/).
+
+
```
Comment (ryanrolds) [inline README.md]: Please remove the extra two blank lines here.

**Pull request merged**